addons/iap/models/ir_config_parameter.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    def _neutralize_iap_service(self, key):
+        if self.get_param(key):
+            neutralized_value = 'https://iap-services-test.odoo.com'
+            param = self.set_param(key, neutralized_value)
+            _logger.info('Neutralized %r from %r to %r', key, param, neutralized_value)
```

Comment:
There's a potential problem of semantics with this: if the param is unset, IAP modules will usually default to a *production* endpoint. So we're resetting IAP params to a test endpoint (which I'm not sure is correct for all of them) if they've been set by the database administrator, but we're leaving in production endpoints otherwise.

Shouldn't this set the param in every case to ensure the service is neutralized?

Reply:
Also wrt IAP, isn't the first thing we should unset / reset / ... the account tokens?

Reply:
And IAP should probably reset its own `iap.endpoint` config parameter.